app/javascript/packs/doc-capture-polling.ts

```diff
@@ -4,6 +4,9 @@ new DocumentCapturePolling({
   statusEndpoint: document
     .querySelector('[data-status-endpoint]')
     ?.getAttribute('data-status-endpoint') as string,
+  phoneQuestionAbTestBucket: document
+    .querySelector('[data-phone-question-ab-test-bucket')
```

Comment:
As mentioned in https://github.com/18F/identity-idp/pull/9481#discussion_r1376745597, I wouldn't expect this will ever match anything, since the selector is missing the closing square bracket `]`.

```suggestion
    .querySelector('[data-phone-question-ab-test-bucket]')
```